ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Public/AccessibilityStatement.cshtml

```diff
@@ -9,88 +9,111 @@
 <div class="govuk-grid-row">
 	<div class="govuk-grid-column-two-thirds">
 		<h1 class="govuk-heading-xl">
-			Accessibility statement for the Manage an academy conversion service
+			Accessibility statement 
 		</h1>
-		<p class="govuk-body">
-			The Manage an academy conversion service is run by the Service Design and
-			Delivery team at the Department for Education (DfE).
-		</p>
-		<p class="govuk-body">
-			We're committed to enabling you to use this service.
+
+		<p class="govuk-body-m">
+			This accessibility statement applies to Prepare conversions and transfers. 
 		</p>
-		<p class="govuk-body">
-			It's designed so that you can:
+		<p class="govuk-body-m">
+			This website is run by the Department for Education. We want as many people as possible to be able to use this website. For example, that means you should be able to: 
```

Comment:
This doesn't quite read right in my head.
"For example, that means you should be able to" 
Its not a big issue because the meaning still comes across.

Reply:
Thanks Jordan. This is more of a content stuff and I'd rather deploy it and let Ady revisit it again if necessary